test_scripts/Defects/5_1/3196_SDL_responds_resultCode_GENERIC_ERROR_while_dataType_VEHICLEDATA_EXTERNTEMP_is_VEHICLE_DATA_NOT_AVAILABLE_and_not_in_subscribed_list_store.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@HSavynetska Please do the following:
1) move scripts to `6_1` folder
2) create separate test set for it, since it was not in scope of 4x-5x

Reply:
Hi @dboltovskyi . Your comment was processed.